test/cell_test.rb

```diff
@@ -52,32 +52,28 @@ def test_fire_upon_updates_fired_upon_to_true
   end
 
   def test_it_displays_a_dot_if_it_has_not_been_fired_upon
+    @cell.place_ship(@ship)
```

Comment:
According to the spec, we should not need to have a Ship inside of the Cell to display a `dot`.

Reply:
@lpile

Reply:
I see, I will try something else

Reply:
```def render(boolean = false)
    if boolean == true
      "S"
    elsif @fire_upon == false
      "."```
Yes! This works! Great stuff!